hooks/tk-multi-loader2/tk-maya_actions.py

```diff
@@ -496,3 +523,37 @@ def _assign_shaders_to_objects(self, src_mtl_mapping, ref_node):
                         cmds.sets(mesh_name, e=True, forceElement=shader)
                     except:
                         print "%s Shader assignment failed for %s" % (shader, mesh_name)
+
+    def _create_standin_reference(self, path, sg_publish_data):
```

Comment:
If there is no difference, can we just use the "create_reference" action? There is no need for a separate action here.